jrt/src/com/yahoo/jrt/TransportThread.java

```diff
@@ -294,7 +301,7 @@ private void run() {
 
             // perform I/O selection
             try {
-                selector.select(100);
+                selector.select(SELECT_TIMEOUT_MS);
```

Comment:
I think this should only be changed in order to improve the accuracy of timeouts. Then we would also need to change the scheduler tick size. C++ uses 10ms epoll timeout and scheduler tick size. We could do that in Java as well, but in a separate pull request. Note that using ScheduleNow when we get a reply will result in the task being run right afterwards (independent of select timeout). As far as I can see, the changes to Mirror.java will make the select timeout only relevant for the back-off case where we use an actual timeout value.